ISISComputingGroup/EPICS-ioc
# Changes to improve the display and interaction for current and voltage

### Description of work

Updates to the current and voltage display to allow users to see what the expect to see

### To test

Needs a change to the Danfysik support module, and danfysiks of various types, or check out NDEMUONFE where this changes are already in place

Part of https://github.com/ISISComputingGroup/IBEX/issues/1597

### Acceptance criteria

These changes are already in place and working, so are covering the appropriate acceptance criteria, which is basically that instead of showing for example a CURR of 3009 and voltage of 63 from a Danfysik 9100 it can instead be shown as 30.09 and 6.3 using the macros, and the current can be set in the same units. The change for the 8800 relating to the command for the current means that a suitable value can be obtained.

---

#### Code Review

- [ ] Is there an [emulator](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Emulating-Devices) for the device? - Note there is currently no emulator, and given the complexities and that these are fixes for a live system I have not put the time in as part of this ticket

### Functional Tests


### Final steps
